### DISCUSSION TITLE
Unscented Kalman Filter for non-linear systems

### DISCUSSION THREAD

**Discussion by raul-parada** [alan-turing-institute/sktime#3705] [Ideas]
Hi,

I don't know if the current KalmanFilterTransformerFP and KalmanFilterTransformerPK functions are for non-lineal systems. Is there a plan to introduce a non-linear model such as Unscented Kalman Filter?

Thanks.

**Comment by fkiraly** [comment]
FYI @NoaBenAmi, @lielleravid
